pom.xml

```diff
@@ -48,6 +48,11 @@
             <version>1.4</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
```

Comment:
not required, just forgot to remove it (for some reason i couldn't execute the tests without this explicit dependency)